Changes needed in server.cfg, mainly :
    Making use of RabbitMQ (default is ActiveMQ)
    The default port used to connect to RabbiMQ is 61613
    Default user : 'mcollective' & passwd : 'changeme'
NOTE : This patch is developed in-house (and Solaris specific)

--- ORIG/etc/server.cfg.dist	2016-02-23 13:46:36.467260235 +0000
+++ mcollective-2.8.2/etc/server.cfg.dist	2016-02-24 17:17:34.551817568 +0000
@@ -1,7 +1,8 @@
 main_collective = mcollective
 collectives = mcollective
-libdir = /usr/libexec/mcollective
-logfile = /var/log/mcollective.log
+libdir = /usr/share/mcollective
+logger_type = file
+logfile = /dev/stderr       # Send output to SMF service log
 loglevel = info
 daemonize = 1
 
@@ -9,12 +10,13 @@
 securityprovider = psk
 plugin.psk = unset
 
-connector = activemq
-plugin.activemq.pool.size = 1
-plugin.activemq.pool.1.host = stomp1
-plugin.activemq.pool.1.port = 6163
-plugin.activemq.pool.1.user = mcollective
-plugin.activemq.pool.1.password = marionette
+connector = rabbitmq
+plugin.rabbitmq.vhost = /mco
+plugin.rabbitmq.pool.size = 1
+plugin.rabbitmq.pool.1.host = localhost
+plugin.rabbitmq.pool.1.port = 61613
+plugin.rabbitmq.pool.1.user = mcollective
+plugin.rabbitmq.pool.1.password = changeme
 
 # Facts
 factsource = yaml
